vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionManager.java

```diff
@@ -66,4 +79,42 @@ public IExhibition getStartExhibition() {
         }
         return null;
     }
+
+    /**
+     * Updates the Exhibition with given list of languages. It fetches the language from exhibitionLanguageConfig using code.
+     *  
+     * @param exhibition
+     * @param defaultLanguage 
+     * @param languages
+     */
+    @Override
+    public void updateExhibitionLanguages(Exhibition exhibition, List<String> codes, String defaultLanguage) {
+        if(CollectionUtils.isNotEmpty(codes)) {
+            if(defaultLanguage!=null) {
```

Comment:
`&& !codes.contains(defaultLanguage)` or you might end up with the same code twice?